Assignment8/Assignmnet8/MainWindow.xaml.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.Win32;
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Threading;
+
+
+namespace Assignmnet8
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private readonly DispatcherTimer _DispatcherTimer = new();
+        private DateTime _TimeAtStart = DateTime.Now;
+        private int Loop = 1;
+        private string _Details;
+        private bool _StartStop = true;
+
+        public MainWindow()
+        {
+            InitializeComponent();
+
+            _DispatcherTimer.Tick += new EventHandler(DispatcherTimer_Tick);

```

Comment:
Event handler can be simplified.
```suggestion
            _DispatcherTimer.Tick += DispatcherTimer_Tick;

```